sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioConfigReader.scala

```diff
@@ -27,7 +27,14 @@ import scala.io.BufferedSource
 class AlluxioConfigReader {
 
   def readAlluxioMasterAndPort(conf: RapidsConf): (String, String) = {
-    readMasterAndPort(conf.getAlluxioHome)
+    if(conf.getAlluxioMaster.isEmpty) {
+      // Read from local Alluxio home
+      readMasterAndPort(conf.getAlluxioHome)
+    } else {
+      // Alluxio master separately deployed and not co-located with Spark Driver.
+      // Like: EKS Env,
+      (conf.getAlluxioMaster, "19998")
```

Comment:
seems odd to assume port is 19998, we should either add another config for port as well

Reply:
Done